server/src/main/java/org/opensearch/action/admin/cluster/repositories/put/PutRepositoryRequest.java

```diff
@@ -78,7 +78,7 @@ public PutRepositoryRequest(StreamInput in) throws IOException {
         settings = readSettingsFromStream(in);
         verify = in.readBoolean();
 
-        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+        if (in.getVersion().onOrAfter(Version.V_2_10_0)) {
```

Comment:
do we need to do same in `server/src/main/java/org/opensearch/cluster/metadata/RepositoryMetadata.java` and `server/src/main/java/org/opensearch/OpenSearchServerException.java` as well